Ecosystem Partners/getting-started-with-vormetric-dsm.md

```diff
@@ -0,0 +1,103 @@
+
+
+### Audience
+CenturyLink Cloud Users
+
+### Impact
+After reading this article, the user should feel comfortable deploying the Vormetric DSM technology on CenturyLink Cloud.  Vormetric has provided a Virtual Appliance - what we call a Partner Template - that can be deployed to your CenturyLink Cloud account via a Service Task.  
+
+This deployment process for Partner Templates currently requires manual interaction via the Service Task process, but will be further automated in future releases of the CenturyLink Cloud Platform.
+
+
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user.
+- Identify a Network VLAN you want the Vormetric DSM to reside on
+
+### Postrequisite
+- If you want to access your Vormetric DSM over the internet, please perform the following tasks once your Vormetric DSM has been deployed to your account:
+1. Adding an external ip-address to an existing or new CenturyLink server
+![ip.jpg](https://t3n.zendesk.com/attachments/token/kObGC9P2IjP1ate0NexwFNiXz/?name=ip.jpg)
+
+2. Allow incoming traffic for port 5666 to an existing or new server
+![port.jpg](https://t3n.zendesk.com/attachments/token/1Ufw0JjIWW8XfASYLh4x3Irl9/?name=port.jpg)
+
+### Detailed Steps to Deploy Vormetric DSM
+Vormetric DSM deploys in a virtual appliance model, as a CenturyLink Cloud *Partner Template*.  Follow these step by step instructions to deploy a Vormetric DSM in to your CenturyLink Cloud account:  
+1. Open a service task request ticket via email to ServiceTasks@Tier3.com with the following details.  You will need to edit some of the information below.
+
+----
+>TO: ServiceTasks@Tier3.com
+>
+EMAIL SUBJECT:   Ecosystem Partner Template Import Request
+>
+CLC Support Team,
+Please create a ticket to import the Ecosystem Partner Template image  referenced below to my CenturyLink Cloud Account:
+- Import CenturyLink Ecosystem Partner Source Image: Vormetric DSM
+- My CenturyLink Cloud Account Alias: ####
+- Data Center to import image to: ###
+- Server Name to import image as: ##########
+- VLAN in the account to add the Server to: ########
+- Additional Notes or work to be done: None
+>
+Please let me know if you have any questions or issues. Kindly send me a reply once the work has been completed and let us know the IP address of the server where this technology has been deployed.
+>
+Thank you very much,
+>
+Your_Name_Here
+
+-----
+
+If you are interested in seeing this type of Partner Template deployment as an automated feature in the future, please share your input with us at [features@centurylinkcloud.com](mailto:features@centurylinkcloud.com)
+
+### Pricing
+The costs associated with this deployment is $195/hr for the CenturyLink Cloud Service Tasks.  There are no Vormetric license costs or additional fees bundled in.  Service Task Fees are available here: http://www.centurylinkcloud.com/service-tasks#Pricing
+
+### Frequently Asked Questions
+Frequently Asked Questions:
```

Comment:
Format for FAQ seems messed up. The second "Frequently Asked Questions:" is redundant.